webpack_config/makeConfig.js

```diff
@@ -295,11 +296,11 @@ module.exports = function(opts = {}) {
     filename: options.isProduction ? '[name].[chunkhash:8].js' : '[name].js',
     publicPath: isDownloadable && options.isProduction ? './' : '/',
     crossOriginLoading: 'anonymous'
-  }
-
+  };
 
   // The final bundle
   return {
+    devtool,
```

Comment:
For anyone else reviewing: This is the only non-formatting change.

Can't believe I missed this D:

Reply:
ah yeah, looks like we dont have prettier setup for things outside of `common`